yt_dlp/extractor/crunchyroll.py

```diff
@@ -158,39 +185,36 @@ def _extract_formats(self, stream_response, display_id=None):
         else:
             full_format_langs = set(map(str.lower, available_formats))
 
-        audio_locale = traverse_obj(stream_response, ((None, 'meta'), 'audio_locale'), get_all=False)
+        audio_locale = stream_response.get('audioLocale')
```

Comment:
maybe not necessary, but with how unpredictable the crunchy api can be, safe traversal and expected types can't hurt
```suggestion
        audio_locale = traverse_obj(stream_response, ('audioLocale', {str}))
```